docs/podman-run.1.md

```diff
@@ -809,18 +811,21 @@ Set the UTS mode for the container
 
 **NOTE**: the host mode gives the container access to changing the host's hostname and is therefore considered insecure.
 
-**--volume**, **-v**[=*[HOST-DIR:CONTAINER-DIR[:OPTIONS]]*]
+**--volume**, **-v**[=*[HOST-DIR-OR-VOUME-NAME:CONTAINER-DIR[:OPTIONS]]*]
 
 Create a bind mount. If you specify, ` -v /HOST-DIR:/CONTAINER-DIR`, podman
 bind mounts `/HOST-DIR` in the host to `/CONTAINER-DIR` in the podman
-container. The `OPTIONS` are a comma delimited list and can be:
+container. Similarly, `-v VOLUME-NAME:/CONTAINER-DIR` will mount the volume 
+to the container. If no such named volume exists, podman will create one.
```

Comment:
a couple of "podman" instances that I'd prefer as "Podman".  One was pre-existing.

Reply:
> a couple of "podman" instances that I'd prefer as "Podman". One was pre-existing.

You mean it's better to substitute "podman" in the text to "Podman"? Other places in the documentation use "podman". If you think that's better, maybe we can replace all of them.

Reply:
If you've the time, then please do change all 'podman" instances to 'Podman' unless it's in a command example.